### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by YegorKochetkov**

### CODE REVIEW

**Review by IlliaBezditnyi — APPROVED** [review]
looks good

File: src/calculateRentalCost.js [hunk 1]
```diff
@@ -29,6 +29,17 @@
...
   // write code here
+  const carCosts = 40;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+
+  if (days >= 7) {
+    return carCosts * days - bigDiscount;
+  } else if (days >= 3) {
+    return carCosts * days - smallDiscount;
+  }
```
Comment (Anna-Tkachenko) [inline src/calculateRentalCost.js:40]: Check point 3 in checklist

File: src/calculateRentalCost.js [hunk 2]
```diff
@@ -29,6 +29,17 @@
...
+
+  return carCosts * days;
```
Comment (Anna-Tkachenko) [inline src/calculateRentalCost.js:42]: Don't repeat yourself: create separate variable for carCosts * days value

**Review by Anna-Tkachenko — CHANGES_REQUESTED** [review]

**Review by Anna-Tkachenko — APPROVED** [review]
Good job 👍

**Pull request closed without merging**